### ISSUE TITLE
Drop support of PHP 5

### ISSUE THREAD

**Issue by llaville** [overtrue/phplint#155]
PHP 5 is no more supported officially since a very long time now.

- PHP 7.x is supported through branch `7.4`
- PHP 8.0 is supported through branch `8.0`
- PHP 8.1 is supported through branch `8.1`
- PHP 8.2 will be supported through branch `8.2`

/cc @overtrue are you agree to drop PHP 5 support on branch 7.4 ?

**Comment by llaville** [comment]
@overtrue Awaiting your agreement ...

**Comment by overtrue** [comment]
oh, I am sorry to miss this notification, just do it.

**Comment by llaville** [comment]
Will keep support PHP 7.0 to 7.3 even if not maintained anymore.  Command `composer why php` show us 

```
overtrue/phplint                          7.4.x-dev requires php (^7.0)
brainmaestro/composer-git-hooks           v2.8.5    requires php (^5.6 || >=7.0)
composer/pcre                             1.0.1     requires php (^5.3.2 || ^7.0 || ^8.0)
composer/semver                           3.3.2     requires php (^5.3.2 || ^7.0 || ^8.0)
composer/xdebug-handler                   2.0.5     requires php (^5.3.2 || ^7.0 || ^8.0)
doctrine/annotations                      1.13.3    requires php (^7.1 || ^8.0)
doctrine/lexer                            1.2.3     requires php (^7.1 || ^8.0)
friendsofphp/php-cs-fixer                 v2.19.3   requires php (^5.6 || ^7.0 || ^8.0)
php-cs-fixer/diff                         v1.3.1    requires php (^5.6 || ^7.0 || ^8.0)
php-parallel-lint/php-console-color       v1.0.1    requires php (>=5.3.2)
php-parallel-lint/php-console-highlighter v1.0.0    requires php (>=5.3.2)
psr/cache                                 1.0.1     requires php (>=5.3.0)
psr/container                             1.1.2     requires php (>=7.4.0)
psr/event-dispatcher                      1.0.0     requires php (>=7.2.0)
psr/log                                   1.1.4     requires php (>=5.3.0)
symfony/console                           v5.4.14   requires php (>=7.2.5)
symfony/deprecation-contracts             v2.5.2    requires php (>=7.1)
symfony/event-dispatcher                  v5.4.9    requires php (>=7.2.5)
symfony/event-dispatcher-contracts        v2.5.2    requires php (>=7.2.5)
symfony/filesystem                        v5.4.13   requires php (>=7.2.5)
symfony/finder                            v5.4.11   requires php (>=7.2.5)
symfony/options-resolver                  v5.4.11   requires php (>=7.2.5)
symfony/polyfill-ctype                    v1.26.0   requires php (>=7.1)
symfony/polyfill-intl-grapheme            v1.26.0   requires php (>=7.1)
symfony/polyfill-intl-normalizer          v1.26.0   requires php (>=7.1)
symfony/polyfill-mbstring                 v1.26.0   requires php (>=7.1)
symfony/polyfill-php70                    v1.20.0   requires php (>=7.1)
symfony/polyfill-php72                    v1.26.0   requires php (>=7.1)
symfony/polyfill-php73                    v1.26.0   requires php (>=7.1)
symfony/polyfill-php80                    v1.26.0   requires php (>=7.1)
symfony/process                           v5.4.11   requires php (>=7.2.5)
symfony/service-contracts                 v2.5.2    requires php (>=7.2.5)
symfony/stopwatch                         v5.4.13   requires php (>=7.2.5)
symfony/string                            v5.4.14   requires php (>=7.2.5)
symfony/yaml                              v5.4.14   requires php (>=7.2.5)
```

**Comment by llaville** [comment]
Available in latest release 3.3.1
